universum/modules/structure_handler.py

```diff
@@ -78,11 +78,10 @@ def is_successful(self) -> bool:
 class StructureHandler(Module):
     def __init__(self, *args, **kwargs):
         super(StructureHandler, self).__init__(*args, **kwargs)
-        block_structure = Block("Universum")
```

Comment:
I'm not sure what was the purpose of this variable

Reply:
It was added to improve readability, but I think it can be removed now.